tests/functional/cylc-message/00-ssh.t

```diff
@@ -21,16 +21,13 @@
 export REQUIRE_PLATFORM='loc:remote comms:ssh'
 . "$(dirname "$0")/test_header"
 #-------------------------------------------------------------------------------
-set_test_number 3
+set_test_number 2
 install_workflow "${TEST_NAME_BASE}" "${TEST_NAME_BASE}"
 
 run_ok "${TEST_NAME_BASE}-validate" \
     cylc validate "${WORKFLOW_NAME}"
 workflow_run_ok "${TEST_NAME_BASE}-run" \
     cylc play --debug --no-detach --reference-test "${WORKFLOW_NAME}"
 
-grep_ok "\[client-command\] graphql ssh" \
-    "$RUN_DIR/${WORKFLOW_NAME}/log/workflow/log" 
-
```

Comment:
This was a safety test to ensure that the comms method is logged.

Perhaps we could add a Cylc client command into the workflow e.g. `cylc hold $CYLC_WORKFLOW_ID//1/elephant` which should give us a `[command]` entry that we can check.

Reply:
Yes, are you happy if I do this in the ssh async request PR as this test is broken anyway?